study-builder/studies/study-example.conf

```diff
@@ -79,10 +79,11 @@
   ],
 
   # The list of languages supported by a study, the order is unimportant.
-  # Each language is a 2-letter ISO language code, e.g. "en", "ru", "fr" etc and set isDefault to true/false
+  # Each language is a 2-letter ISO language code, e.g. "en", "ru", "fr" etc , language name e.g. "English" and isDefault (true/false)
   # Optional, can be left out or set to null.
   "supportedLanguages": [
       "language": "string",
+      "name": "string",
       "isDefault": bool
```

Comment:
should we enforce to set atleast one language as default if supportedLanguages are set !!!

Reply:
I think we should. In some areas of codebase, we try to fallback to the study's default language. And for frontend clients, they will need to know which is the study's default language.